### PR TITLE
arash/ Card component

### PR DESCRIPTION
**Pull request by arash-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

Change the card component UI and add a temporary card with the old ul for the home page

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/other-platforms.js [hunk 1]
```diff
@@ -44,9 +44,9 @@ const StyledSubHeader = styled(Header)`
     max-width: ${props => props.maxWidth || ''};
 `
 
-export const OtherPlatform = ({ header, subHeader, exclude }) => (
-    <SectionContainer>
-        <HeaderWrapper>
+export const OtherPlatform = ({ header, subHeader, exclude, nav }) => (
+    <SectionContainer padding='0'>
+        {nav ? null : (<HeaderWrapper>
```
Comment (mustofa-binary) [inline src/components/custom/other-platforms.js]: is it to check if its a nav?, if it is, please change the name to is_nav

File: src/components/custom/platforms-dropdown.js [hunk 2]
```diff
@@ -42,96 +38,22 @@ const NavDropdown = styled.div`
...
-                                {localize('DMT5')}
-                            </Text>
-                            <Text color="black-3">
-                                {localize('Trade with the platform of choice for professionals.')}
-                            </Text>
-                        </div>
-                    </PlatformItem>
-                </NavDropdownGrid>
+                <StyledContainer>
+                    <OtherPlatform exclude="" nav/>
```
Comment (mustofa-binary) [inline src/components/custom/platforms-dropdown.js]:
```suggestion
                    <OtherPlatform is_nav/>
```

File: src/components/elements/card.js [hunk 3]
```diff
@@ -83,50 +120,65 @@ const IconContainer = styled.div`
...
                 <>
                     {is_inline_icon ? (
                         <>
+                            <CardCover background_color={cover_background}>
+                                <div>
+                                    <h4>
+                                        {cover_content}
+                                    </h4>
+                                    <h4>
+                                        &gt;
```
Comment (mustofa-binary) [inline src/components/elements/card.js]: can please add comment on what symbol is this

File: src/components/custom/other-platforms.js (hunk 1 above)
Comment (arash-fs) [reply src/components/custom/other-platforms.js]: done

File: src/components/custom/platforms-dropdown.js (hunk 2 above)
Comment (arash-fs) [reply src/components/custom/platforms-dropdown.js]: done

File: src/components/elements/card.js (hunk 3 above)
Comment (arash-fs) [reply src/components/elements/card.js]: done

**Pull request merged**